scripts/validate_string_concatenation.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
```

Comment:
Is this correct? Might be what's causing your issue mentioned https://github.com/pandas-dev/pandas/pull/30467#issuecomment-569092377